addons/website/static/src/js/utils.js

```diff
@@ -4,7 +4,7 @@ odoo.define('website.utils', function (require) {
 var ajax = require('web.ajax');
 var core = require('web.core');
 
-var qweb = core.qweb;
```

Comment:
You're trying to extract `qweb` and `_t` from `core.qweb`. This line should be

```js
const { qweb, _t } = core;
```

This is what's causing CI to fail (_t and qweb are undefined)